azerothcore/azerothcore-wotlk
# fix(Core/Events): Corrections to Apothecary Hummel

## Changes Proposed:
-  Removed old EventMap implementation
-  Added missing phase.
-  Despawn Crazed Apothecary adds on reset.

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/3037

## SOURCE:


## Tests Performed:

-  Built but not tested.


## How to Test the Changes:


1. Start event love is in the air.
2. Go to Apothecary Hummel and wipe.
3. It should restart properly.
